lib/snail.rb

```diff
@@ -35,6 +35,7 @@ class UnknownAttribute < ArgumentError; end
     :locality   => :city,
     :state      => :region,
     :province   => :region,
+    :county     => :region,
```

Comment:
Interesting! But would this mean that the term county is ambiguous? It's definitely not equivalent to state and province in the US.